Source/Documentation/Manual/features-rollingstock.rst

```diff
@@ -318,6 +318,79 @@ for the purposes of calculation.
 - ``MU ( 3 )`` Locomotive must be in a different group to the lead locomotive
 
 
+Multiple type locomotive light glows
+------------------------------------
+
+Introduction
+''''''''''''
+
+As a default all OR (and MSTS) locomotives use the same texture to reproduce 
+the glow of their lights. This however doesn't allow to easily implement non-round 
+or LED array lights, neither to provide special glowing effects.
+
+This feature allows to specifiy customized light glow textures for the locomotives. 
+If nothing is specified, the standard light glow texture is used. Moreover in the ``Content`` 
+folder, two light glow textures are present: the "historical" one, and a new one, 
+more realistic. As a default the "historical" light glow texture is used, for backwards 
+compatibility; however adding a line to the Lights block in the .eng file the "new" light 
+glow texture is taken. Customized light glow textures can be either used for all lights 
+of a loco, or only for a subset of them. Different lights in the same locomotive 
+may have different customized light glow textures.
+
+Detailed spec
+'''''''''''''
+
+
+1) In the ``Content`` folder there is the default ``LightGlow.png``, which is displayed if 
+   no changes are done to the .eng file.
+2) In such folder there is also an ``ORTSLightGlow.png``, which is maybe more realistic.
+3) adding a line within the .eng file it is possible to select either ORTSLightGlow.png or any other .png. 
```

Comment:
I tried using a .ace texture and that actually worked. However, _.dds does not work_. This seems to be a slight oversight in Materials.cs line 180, which is a backup case for loading .ace but was never updated to consider .dds support. We should probably support all the texture formats used in the MSOR ecosystem.

In any case, the manual should not imply that .png is the only allowed format. Currently, .png, .jpg, .gif, .bmp, and .ace are supported. That's pretty much every common type of image file (and the MSTS format, of course), so all file types should be listed out, or described shorthand "all standard image formats are supported".

Reply:
Done

Reply:
I don't foresee to add light rotation and rectangular lights. This goes quite beyond the initial aim of this PR. Such additions could be object of a further PR.